ds/data/processed/eurostat/epo_patent_applications.nuts2.yaml

```
@@ -28,7 +28,7 @@ schema:
 source_name: Eurostat
 source_url: https://ec.europa.eu/eurostat
 warning: The regional breakdown of the EU Member States is based on the Nomenclature of Territorial Units for Statistics – NUTS 2010 level 3
-year_range: [1996,2016]
+year_range: [1996,2012]
```

According to the data I think it should be

```suggestion
year_range: [2006,2012]
```